### PR TITLE
terraform-helm-keycloak

### PR DESCRIPTION
**Pull request by volsus83**
This Pull Request will provide a Terraform description for Keycloak deployment on Azure cloud platform

### CODE REVIEW

File: deployment/terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+git clone https://github.com/eclipse/kuksa.cloud.git
+cd kuksa.cloud/deployment/terraform
+
+# login to Azure and set subscription
+az login
+az account set --subscription <ID>
+
+# prepare infrastructure configuration
+cp ./terraform.tfvars.template ./terraform.tfvars
+code ./terraform.tfvars
```
Comment (unly) [inline deployment/terraform/README.md]: this is misleading. why open vs code? needs a comment regarding what to change. if an editor is needed maybe switch to a more generic that is more likely already installed such as `vi`

File: deployment/terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+# deploy infrastructure
+source ./prepare_environment.sh
+terraform workspace select <dev, stage or prod>
+terraform init -upgrade
+terraform plan
```
Comment (unly) [inline deployment/terraform/README.md]: apply prints a plan anyway

File: deployment/terraform/README.md (hunk 1 above)
Comment (volsus83) [reply deployment/terraform/README.md]: You're right, I will change this. I have no intention to give any editor recommendation.

File: deployment/terraform/README.md (hunk 2 above)
Comment (volsus83) [reply deployment/terraform/README.md]: For current versions of Terraform this is correct. This how-to should be independent of Terraform version but raise the claim to list all steps you should do even if you use "-auto-approve" flag regularly or in automation.

**Comment by eriksven** [comment]
Thank you for the PR. We will merge this once we are done with the work for the 0.1.0 Release.

**Comment by bs-jokri** [comment]
This PR contains a deplyoment for an outdated vrsion of keycloak. We should update as described here https://github.com/eclipse/kuksa.cloud/issues/121

**Pull request merged**